addons/web_editor/static/src/js/wysiwyg/wysiwyg.js

```diff
@@ -296,7 +296,11 @@ const Wysiwyg = Widget.extend({
                     })();
                 }
                 $target.focus();
-                if ($target.closest('#wrapwrap, .iframe-editor-wrapper').length) {
+                // If the link contains a media without text, the link is
+                // editable in the media options. Otherwise it must be
+                // configured using the link tools.
+                if ($target.closest('#wrapwrap, .iframe-editor-wrapper').length
+                        && !($target[0].querySelector(mediaSelector) && !$target[0].textContent.trim())) {
```

Comment:
I rebased and push-forced because there were some conflict with 15.0 so don't forget to fetch first before chaning anything.

I guess it's ok if no other way but I am not sure that this wysiwyg.js file should suppose anything about "snippet" options like you do here. I'd prefer that `toggleLinkTools` is called but somehow did nothing hurtful if link tools are not available for this specific DOM. Could you check? Thanks

In fact, that's true that the `activate_image_link_tool` event I asked you to use already breaks that concept... but if we can somehow not worsen the issue? If not, tell me, I am fine to fix it that way first and leave that to a master refactoring :grimacing: Maybe there should be some test though?

Reply:
I moved the condition within `toggleLinkTools` and added steps inside the existing link tools tour to verify the behavior.